tensorcircuit/templates/lattice.py

```diff
@@ -1477,24 +1695,31 @@ def get_compatible_layers(bonds: List[Tuple[int, int]]) -> List[List[Tuple[int,
         tuple represents a bond. All bonds within a layer are non-overlapping.
     :rtype: List[List[Tuple[int, int]]]
     """
-    uncolored_edges: Set[Tuple[int, int]] = {(min(bond), max(bond)) for bond in bonds}
+    logger.debug(f"[DEBUG-LATTICE] Getting compatible layers for {len(bonds)} bonds.")
+    # Ensure all bonds are in a canonical form (i, j) with i < j and remove duplicates.
+    sorted_edges = sorted(list({(min(bond), max(bond)) for bond in bonds}))
 
     layers: List[List[Tuple[int, int]]] = []
+    unassigned_edges = set(sorted_edges)
```

Comment:
what is the point of the renaming, which can induce more than 10 lines of diff with no diff at all